test/integration/ipamperf/test-performance.sh

```diff
@@ -20,27 +20,49 @@ set -o pipefail
 
 TEST_ARGS=""
 RUN_PATTERN=".*"
+PROFILE_OPTS=""
 
 function usage() {
   echo "usage: $0 [-h] [-d] [-r <pattern>] [-o <filename>]"
   echo " -h display this help message"
   echo " -d enable debug logs in tests"
   echo " -r <pattern> regex pattern to match for tests"
   echo " -o <filename> file to write JSON formatted results to"
+  echo " -p <id> enable cpu and memory profiles, output written to mem-<id>.out and cpu-<id>.out"
```

Comment:
seems easier to pass args after `--` directly to the test executable so we don't need to redo the options on each layer

Reply:
The reason I exposed this is because the current arguments today are interpreted as a mix of golang args and test specific args passed down the code. Wanted to retain the ease of use and management of these details in the runner.

If we moved to direct args, then I feel it would be better to go test -c and then run the binary with the args directly. But I don't feel we need to make that change just yet.

But it did remind me that I have not updated the README for the custom test configuration, did that and pushed the new changes as part of this PR.